config/prod/dian-hm-prod-migration-s3.yaml

```diff
@@ -0,0 +1,42 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: "remote/s3-bucket-v2.yaml"
```

Comment:
this is an old template.  the latest s3 bucket template is here: https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/s3-bucket-v2.j2

Reply:
Ok, updated to use newest template